.gitignore

```diff
@@ -8,3 +8,4 @@ main.php
 config.json
 auth.json
 tools
+/composer.lock
```

Comment:
`composer.lock` deliberately committed here, even though this is a library.

Reply:
So thing is, composer installs different packages depending on the version of PHP. Should the later versions of php not install newer versions of the library? Leaving the composer.lock makes one of the tests error either lowest or highest. What's the reason to deliberately commit?

Reply:
We keep the composer.lock committed specifically so that the lowest tests will install different libraries than the locked tests, and the highest tests will install different libraries still. This insures that the different versions of Symfony that we claim to work with actually continue to work.

Reply:
Yeah, those old symphony versions are triggering the dependeabot alert because they require an older version of guzzle.